scripts/CYAC-JP/c101112071.lua

```diff
@@ -0,0 +1,83 @@
+--烙印の即凶劇
+--Script by 奥克斯
+function c101112071.initial_effect(c)
+	--activate
+	local e0=Effect.CreateEffect(c)
+	e0:SetType(EFFECT_TYPE_ACTIVATE)
+	e0:SetCode(EVENT_FREE_CHAIN)
+	c:RegisterEffect(e0)
+	--Effect 1
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(101112071,0))
+	e1:SetCategory(CATEGORY_SPECIAL_SUMMON)
+	e1:SetType(EFFECT_TYPE_QUICK_O)
+	e1:SetCode(EVENT_FREE_CHAIN)
+	e1:SetRange(LOCATION_SZONE)
+	e1:SetHintTiming(0,TIMINGS_CHECK_MONSTER+TIMING_END_PHASE)
+	e1:SetCountLimit(1,101112071)
+	e1:SetTarget(c101112071.sctg)
+	e1:SetOperation(c101112071.scop)
+	c:RegisterEffect(e1)	 
+	--macro cosmos
+	local e2=Effect.CreateEffect(c)
+	e2:SetType(EFFECT_TYPE_FIELD)
+	e2:SetProperty(EFFECT_FLAG_SET_AVAILABLE+EFFECT_FLAG_IGNORE_RANGE+EFFECT_FLAG_IGNORE_IMMUNE)
+	e2:SetRange(LOCATION_SZONE)
+	e2:SetCode(EFFECT_TO_GRAVE_REDIRECT)
+	e2:SetTargetRange(0xfe,0xff)

```

Comment:
Bug:
```
Branded Etude is banishing my own materials when monsters are used for fusion or synchro summons. . .
. . . Branded Etude is one-sided
```
[Report Source](https://forum.duelistsunite.org/t/branded-etude-bug/15366)
```
While you control a "Bystial" monster, any monster sent to your opponent's GY because they Tributed it for
a Ritual Summon or used it as material for a Fusion, Synchro, or Link Summon is banished instead.
```